test/testLiveAsTemplate.js

```diff
@@ -124,6 +124,10 @@ describe('CLI interface, oghliner as a template', function() {
     .then(function() {
       assert.doesNotThrow(fse.statSync.bind(fse, 'index.html'));
       assert.doesNotThrow(fse.statSync.bind(fse, 'offline-worker.js'));
+    })
+    .catch(function(err) {
+      console.log('Error: ' + err);
+      throw err;
```

Comment:
I've found the problem, opened #174 to fix it and reverted the changes in this branch.